Utilities/Templates/Modules/PythonCLI/PythonCLIModuleTemplate.py

```diff
@@ -0,0 +1,36 @@
+import itk
```

Comment:
See https://pypi.python.org/pypi/flake8-import-order

```
import logging
import sys

from ctk_cli import CLIArgumentParser

import itk
```

Useful resrouces: https://pypi.python.org/pypi/flake8-import-order and